packages/suite/src/views/settings/device/AutoLock.tsx

```diff
@@ -67,7 +67,6 @@ export const AutoLock = ({ isDeviceLocked }: AutoLockProps) => {
             />
             <ActionColumn>
                 <ActionSelect
-                    hideTextCursor
                     useKeyPressScroll
                     placeholder=""
```

Comment:
Can we make this the default value? Otherwise it says "Select..." which does not get translated.

Reply:
a9709de8f17e51925046c6a21f777ecfc66ac31f